_posts/2018-10-01-project-roles.md

```diff
@@ -0,0 +1,81 @@
+
+It became apparent to us, that the roles we take in each project are unclear. There is a huge gray area, especially between team leaders, senior developers and project managers. We haven’t talked about it much so obviously there are some expectations that are not met on both sides.
+
+To challenge this we made a list of all project roles we feel are required, and we did this in two separate teams to combine different points of view. This list was strongly focused on team leader responsibilities first, but it became clear that it’s one sided, and we cannot just dump all this responsibilities on one person, and that it varies from project to project.
+
+## Project roles
+
+What we did instead is a general list of roles, and for now it goes as follows (in random order):
+
+### Helicopter view
+
+Basically knowing more that is required to do your tasks, both in and out of your department. This allows you to connect the dots and make decisions based on the project as a whole. You can’t coordinate tasks and teamwork without a helicopter view. For example, knowing the design progress so you can anticipate and plan accordingly. Or even more trivial example, knowing when one departament will finish their job, so another can start. Or to know when one person can help out another, because they’ve done something similar or their tasks are related somehow.
+
+### Responding to ad-hoc situations
+
+If project is on fire, you’re the first line of defence. You may (and probably need to) delegate it further, but you’re the rapid response team. This allows an easy way out for every team member if something goes wrong and an urgent fix is required, for example the project needs an additional developer, because the project is behind schedule. Without this role, the responsibility would be distributed among the whole team and as such it would be ineffective.
+
+### Reporting
+
+You can present project status to stakeholders and bring anyone up to date. This may relate only to certain fields of the project, such as development, design or other.
+
+### Documenting
+
+Updating knowledge on Confluence, so everyone is on the same page. Documenting has a broader meaning, but for the purpose of defining roles, let’s just stick to gathering all knowledge and arrangements in one place. This may be as trivial as meeting notes — someone has to do it. Without this role, there wouldn’t be a single source of truth about the project; all would be scattered between JIRA tasks, emails, bug trackers, slack, etc.
+
+### Risk Management
+
+This is both bringing awareness of project risks to the team, as well as acting on the current identified risks. For example, if we have a tight schedule, we need to pay extra attention to our estimates, and regular plannings should reflect that, and the team need to be aware of the priorities — no needless refactors this time, sorry. 
```

Comment:
"current identified risks" -> "already identified risks"?

Reply:
"For example..." - za dlugie zdanie, rozbilbym na 2. "and the team need to be aware " zaczalbym jako nowe zdanie w stylu "Moreover, the team needs to be aware "
"team need" -> "team needs"

Reply:
Tutaj jest napisane o prioritetach pod czas fuckupu. Sugeruje że priorytetowanie muśi być oddzielnym punktem.

Reply:
@FrostF0X czy to nie jest planning / project management? Nie chcę się też za bardzo rozdrabniać w tych rolach, ale jak będziemy z Tomkiem o tym rozmawiać, to wspomnę, może trafi na listę